README.md

```diff
@@ -0,0 +1,54 @@
+# Instagram Clone
+
+Instagram clone built with React, Redux, Node.js, Express, Passport.js & MongoDB.
+
+## Features
+
+- Sign up for an account and edit avatar/bio/name later on
+- Upload images with captions and option to edit/delete them
+- Comment on posts and option to edit/delete them
+- Like/unlike posts
+- Follow/unfollow users
+- Feed consists of posts from users you follow and the 'Explore' page has posts from every user
+
+## Features to add
+
+- Demo login button (for people who don't want to register)
+- Add location to posts
+- Add tags to posts
+- Search option for tags and location
+- Bookmark posts
+- Modals that display likes (on a post), and followers/following (on profile)
+
+## Running Locally
+
+The following instructions should get this project running on your machine.  
+
+### Prerequisites
+1. Install [MongoDB](https://www.mongodb.com/)  
+2. Create a [Cloudinary](https://cloudinary.com/) account (need an API key and secret)
+
+### Installing
+```
+git clone https://github.com/101Loop/InstagramClone-BE.git
+cd reactgram
+npm install
+cd client
+npm install
+```
+
+Create a .env file in the root of the project and add the following:
```

Comment:
There is already an keys.js.env 
- you just have to copy that file and rename it to keys.js and then add databaseusername and Password.
- secondly, you have to add a JWT token, which shlould be any random alphabets atleast 14 characters